site/src/sphinx/server-grpc.rst

```diff
@@ -135,7 +136,12 @@ you build a :api:`GrpcService`:
                               .allowRequestMethods(HttpMethod.POST) // Allow POST method.
                               // Allow Content-type and X-GRPC-WEB headers.
                               .allowRequestHeaders(HttpHeaderNames.CONTENT_TYPE,
-                                                   HttpHeaderNames.of("X-GRPC-WEB"));
+                                                   HttpHeaderNames.of("X-GRPC-WEB"))
+                              // Expose trailers of the HTTP response to the client so that it can get
+                              // the their values.
+                              .exposeHeaders(GrpcHeaderNames.GRPC_STATUS,
```

Comment:
I'm curious if I need to use `HttpHeaderNames` instead of `GrpcHeaderNames` because `GrpcHeaderNames` is in the `internal` package. Please let me know what you think!

Reply:
I think we can move it to common. I had it in internal since I thought it was an implementation detail of the wire protocol but didn't know about this sort of use case.

Reply:
Ah, I see. Then, could I move it to common package in this PR? :-)

Reply:
I think so, yeah